models/src/main/scala/coop/rchain/models/rholang/implicits.scala

```diff
@@ -327,7 +324,7 @@ object implicits {
 
     def singleDeployerId(): Option[GDeployerId] =
       if (p.sends.isEmpty && p.receives.isEmpty && p.news.isEmpty && p.exprs.isEmpty && p.matches.isEmpty && p.bundles.isEmpty && p.connectives.isEmpty) {
-        p.unforgeables.toList match {
+        p.unforgeables match {
```

Comment:
I think it would be better to submit a small pr for this kind of fixes

Reply:
@zsluedem All of the commits so far are only preparation and basically clean up with changes like this. It makes sense to extract it to separate PR.
Even more because I didn't find some nice way to extend normalizer to return additional info for debugging so it will probably be just temporary solution not entering main branch.